src/utils/util/requirements.txt

```diff
@@ -1,2 +1,3 @@
 jsonschema
 xmlschema
+ruamel.yaml
```

Comment:
FYI, I am also going to pin specific versions of the package to stay consistent with good practice, @guyzyl. I presume you won't mind! :-)